tests/WebApiGateway.UnitTests/Infrastructure/Builders/Support.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace WebApiGateway.UnitTests.Infrastructure.Builders
+{
+    public static class Support
```

Comment:
Rename to GuidHelper class.
Place file into "Helpers" folder. (WebApiGateway.UnitTests.Infrastructure.Builders.Helpers).